packages/slate-cli/src/utils.js

```diff
@@ -178,3 +197,13 @@ export function isShopifyThemeWhitelistedDir(directory) {
   const whitelist = ['assets', 'layout', 'config', 'locales', 'sections', 'snippets', 'templates'];
   return whitelist.indexOf(directory) > -1;
 }
+
+/**
+ * Tests if file is a Shopify theme settings file (settings_*.json)
+ *
+ * @param {string} file - The path to the file.
+ */
+export function isShopifyThemeSettingsFile(file) {
+  const whitelist = ['settings_schema.json', 'settings_data.json'];
```

Comment:
Maybe for future proof, we can use a Regex to check whether or not the format is `settings_*.json` instead of having a hardcoded whitelist.

Reply:
Could be as simple as that:
```
export function isShopifyThemeSettingsFile(file) {
  return /^settings_.+\.json/.test(file);
}
```

Reply:
Could be as simple as that:
```
export function isShopifyThemeSettingsFile(file) {
  return /^settings_.+\.json/.test(file);
}
```